quarry/web/worker.py

```diff
@@ -148,6 +148,13 @@ def run_query(query_run_id):
             celery_log.info("Stopped run for qrun:%s", qrun.id)
         else:
             write_error(qrun, e.args[1])
+    except pymysql.err.OperationalError as e:
+        qrun.status = QueryRun.STATUS_FAILED
```

Comment:
I think it's also worth surfacing the error message to the user per the comment on line 141. Otherwise, the UI would just say it failed, with no explanation. 

We could just use the write_error() function here.

Reply:
write_error() is just:
```
qrun.status = QueryRun.STATUS_FAILED
    qrun.extra_info = json.dumps({"error": error})
    conn.session.add(qrun)
    conn.session.commit()
    celery_log.info(
        "Completed run for qrun:%s with failure: %s", qrun.id, error
    )
```
which we already do. But I added `qrun.extra_info = json.dumps({"error": e.args[1]})` here too